Replaced abstract placeholder with a concise standfirst summary. (#27)
haesleinhuepf/git-bob-manuscript: git-bob-manuscript.tex

\abstract{

> We present git-bob, an open-source AI-assistant integrated with GitHub/GitLab that facilitates iterative collaboration and transparency in AI-assisted data analysis and code generation within discussion threads.

Reviewer: ```suggestion
How an open-source AI-assistant integrated with GitHub/GitLab can facilitate iterative collaboration and transparency in AI-assisted data analysis and code generation.
```
